.changeset/grumpy-singers-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"functions-subscription": minor
+---
+
+[IOPLT-746] Add listTrial to CosmosDB and http layer
```

Comment:
**suggestion**

```suggestion
[IOPLT-746] Create endpoint to get the trial list
```

what do you think?

Reply:
Updated in acbf3bec67503f6c37e36a4fa82d031c2b8e84ec